docs/reference/objects/product/variant/index.md

```diff
@@ -111,7 +111,7 @@ The variant's [image]({% link docs/reference/objects/image.md %}) if it has one.
 number
 {: .label .fs-1 }
 
-The initial stock for the variant, if the variant has unlimited inventory this will return `nil`.
+The initial stock for the variant. If the variant has been manually marked as 'Sold Out' this will return 0. If the variant has unlimited inventory this will return `nil`.
```

Comment:
I don't think `initial_stock` will be affected by this change, right? Only `remaining_stock`.

Reply:
Ah ha yes of course.